app/javascript/packages/document-capture-polling/index.js

```diff
@@ -68,12 +68,23 @@ export class DocumentCapturePolling {
   }
 
   /**
-   * @param {{ isCancelled: boolean }} params
+   * @param {{
+   *   isCancelled?: boolean,
+   *   isThrottled?: boolean,
+   *   redirect?: string,
+   * }} params
    */
-  async onComplete({ isCancelled }) {
-    await this.trackEvent('IdV: Link sent capture doc polling complete', { isCancelled });
+  async onComplete({ isCancelled = false, isThrottled = false, redirect }) {
```

Comment:
Could also consider a "result" enum, since these are mutually exclusive booleans.

Reply:
enum seems good, esp since we have Typescript checker to help us check?

Reply:
👍  Refactored in 46968ebf2.